### ISSUE TITLE
Cannot start the launcher, no error message at all

### ISSUE THREAD

**Issue by chronowarden** [remyroy/CDDA-Game-Launcher#509]
**Describe the bug**
Can't start the game, same issue as 472:
https://github.com/remyroy/CDDA-Game-Launcher/issues/472

Some older versions work.

**To Reproduce**
Probably can't reproduce without a specific system setup

**Expected behavior**

**Desktop (please complete the following information):**
 - OS: Windows 10 x64

**Additional context**
I have downloaded the Microsoft Visual C++ Redistributable as linked in https://github.com/remyroy/CDDA-Game-Launcher/releases/tag/v1.4.11
I have turned my firewall off.
I have turned my antivirus off.
I have executed as administrator.
Nothing worked.
The debug version as linked in issue #472 does work, neither of the ones from the same version do.

**Comment by remyroy** [comment]
There are a few things that are strange.

The debug version as linked in issue #472 should be crashing on start right now. It has the same issue as the one described in #507 .

Did you have a look in your log file? It should be in `%LOCALAPPDATA%\CDDA Game Launcher\app.log` . There might be some clue in there. Could you send it here?

If there are still some issues, I might be building another debug release for you to test.

**Comment by chronowarden** [comment]
Yeah, correction, the older debug version does not actually work anymore, as you stated, goes unresposive and crashes. I checked the log file, and it seems that the versions that don't actually launch don't even leave logs, I tried opening the 1.4.13 launcher a few times but none of them are in the logs. 
[app.log](https://github.com/remyroy/CDDA-Game-Launcher/files/5541298/app.log)

**Comment by Eldhelion** [comment]
same here, but my log have nothing from today
[app.log](https://github.com/remyroy/CDDA-Game-Launcher/files/5541395/app.log)

**Comment by remyroy** [comment]
I've just built a debug release. You can find it here: https://www.dropbox.com/s/cfcpxaq1fk9yl6t/cddagl_portable_v1.4.13.debug.zip?dl=1

Download this release, start the launcher with launcher-debug.bat. It will create a file called debug.txt. Send that file back to me. It might provide more information regarding your issue to help me fix it.

**Comment by chronowarden** [comment]
The debug launcher succeded in launching the game, here is the debug.txt file.
[debug.txt](https://github.com/remyroy/CDDA-Game-Launcher/files/5554383/debug.txt)
